### ISSUE TITLE
[TD] Error while loading FluidDuct

### ISSUE THREAD

**Issue by MySt1k** [CoFH/Feedback#892]
http://pastebin.com/wed4pq3G

**Comment by KingLemming** [comment]
Error while loading FluidDuct hooks.java.lang.ClassNotFoundException: cofh.thermaldynamics.ducts.fluid.TileFluidDuct

Something apparently broke the mod. Quite obviously, we aren't shipping a broken mod, so something on your end has caused this.

**Comment by MySt1k** [comment]
Whether there are some ways to find out it? I have all standard mods

**Comment by KingLemming** [comment]
Not that I know of. Definitely never seen an error like this one. Redownload the jars and make sure they aren't tampered with.

**Comment by MySt1k** [comment]
changed nothing, I tried to update mods on the old version, but too most

**Comment by MySt1k** [comment]
http://pastebin.com/qfFVr03p

**Comment by MySt1k** [comment]
I removed mod Waila-1.5.10_1.7.10 from the server and the mistake disappeared. Problem obviously with integration, I will report about it to the Waila developers

**Comment by MySt1k** [comment]
It was made to me

https://bitbucket.org/ProfMobius/waila/issues/224/error-while-loading-fluidduct-hooks

**Comment by MySt1k** [comment]
No. The problem is present. Everything exactly crumbles servers.

**Comment by KingLemming** [comment]
I'm sorry, but I don't understand what you're saying here.

You fixed it by removing WAILA, so it's on them? Makes sense, since *we cannot replicate this at all.*

**Comment by MySt1k** [comment]
The problem isn't solved. After I removed Waila a line ([Thermal Dynamics] Error while loading FluidDuct hooks.java.lang.ClassNotFoundException: cofh.thermaldynamics.ducts.fluid.TileFluidDuct) disappeared. I thought crash won't be any more, but the couple of hours back it happened. http://pastebin.com/Ckiubuen

**Comment by KingLemming** [comment]
I'll give this a wtf label for now, but basically what that means is don't expect a resolution.

This doesn't happen to us in dev, nobody else reports it, so it's specific to your install and almost certainly x-mod and/or world corruption. Repeated upgrades or modpack changes could have precipitated this. There's just not a great way to know.

**Comment by MySt1k** [comment]
As I understand, server crash occurs when чанк is loaded and in a pipe there is AIR. A question how it is possible to clean duct forcibly?

**Comment by babyblasphemy** [comment]
Having same issue on our DW20 server, here's a crashlog instead of an excerpt from the server log
http://pastebin.com/khi2jwLV
